keyboards/pteron56/config.h

```diff
@@ -0,0 +1,23 @@
+/*pteron56
+*Copyright (C) 2023 @kraken-jokes
+*
+*This program is free software: you can redistribute it and/or modify
+*it under the terms of the GNU General Public License as published by
+*the Free Software Foundation, either version 3 of the License, or
+*(at your option) any later version.
+*
+*This program is distributed in the hope that it will be useful,
+*but WITHOUT ANY WARRANTY; without even the implied warranty of
+*IMERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+*GNU General Public License for more details.
+*
+*You should have received a copy of the GNU General Public License
+*along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+/* number of backlight levels */
+#ifdef BACKLIGHT_ENABLE
```

Comment:
Does this PCB even have any support for backlight? It is not enabled anywhere nor does it look like any LEDs can be installed from the photos. If the answer is no, this file should be removed. If the answer is yes, this file should still be removed and this setting moved to `info.json`.

Reply:
I am not sure. I removed the file for now.